<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userManagement</class>
 <widget class="QWidget" name="userManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>651</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>671</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>681</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>185</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>275</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>create user</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>mis a jour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
